### ISSUE TITLE
Noto Sans are 100x bigger file size

### ISSUE THREAD

**Issue by caleb87** [openmaptiles/fonts#13]
The Noto Sans fonts are almost 100x bigger than Open Sans. The Noto Sans Regular is 33mb on disk, and Open Sans Regular is 400kb. All the other fonts are tiny, but Noto Sans is enormous. 100mb total. 

Is this normal?

**Comment by erdmann040** [comment]
Yes, this is normal. Noto fonts cover the characters of almost all languages in the world, including Asian languages, which can have many thousands of characters.

**Comment by TomPohys** [comment]
Thanks for the answer.
